addon/src/webextension/lib/browserAction.js

```diff
@@ -34,7 +34,12 @@ export async function setupBrowserAction() {
 
   PubSub.subscribe(environmentTopics('resources'), updateBadgeTextOnNew);
 
+  browser.browserAction.setBadgeBackgroundColor({ color: '#0996f8' });
```

Comment:
Need to bump the [version number in `addon/package.json`](https://github.com/mozilla/testpilot/blob/master/addon/package.json#L5) with any changes to the add-on.